tests/jwk_test.py

```diff
@@ -17,24 +25,36 @@
 class JWKTest(unittest.TestCase):
     """Tests for josepy.jwk.JWK."""
 
-    def test_load(self):
+    def test_load(self) -> None:
         from josepy.jwk import JWK
         self.assertRaises(errors.Error, JWK.load, DSA_PEM)
 
-    def test_load_subclass_wrong_type(self):
+    def test_load_subclass_wrong_type(self) -> None:
         from josepy.jwk import JWKRSA
         self.assertRaises(errors.Error, JWKRSA.load, DSA_PEM)
 
 
+class JWKSubclassTest(Protocol):
+    from josepy.jwk import JWK
+    jwk: JWK
+    thumbprint: bytes
+
+    def assertEqual(self, *unused_args: Any) -> None:
```

Comment:
I suppose this should be removed once rebased with master due to removing `unittest` in e6cea11dea17fa6936576accf878c9a4862fa3b1.

Reply:
Nice catch. I definitely would have missed this.